content/docs/reference-glossary.md

```diff
@@ -1,43 +1,43 @@
 ---
 id: glossary
-title: Glossary of React Terms
+title: React की शर्तों का शब्दकोष
 layout: docs
 category: Reference
 permalink: docs/glossary.html
 
 ---
 
-## Single-page Application {#single-page-application}
+## सिंगल-पेज एप्लीकेशन {#single-page-application}
 
-A single-page application is an application that loads a single HTML page and all the necessary assets (such as JavaScript and CSS) required for the application to run. Any interactions with the page or subsequent pages do not require a round trip to the server which means the page is not reloaded.
+सिंगल-पेज एप्लिकेशन एक ऐसा एप्लिकेशन है जो एक HTML पेज को लोड करता है और आवश्यक सभी संपत्तियो को भी (जैसे जावास्क्रिप्ट और सीएसएस) जो एप्लिकेशन को चलाने के लिए आवश्यक है। पृष्ठ या बाद के पृष्ठों के साथ किसी भी बातचीत को सर्वर के लिए एक गोल यात्रा की आवश्यकता नहीं होती है जिसका अर्थ है कि पृष्ठ पुनः लोड नहीं हुआ है।
 
-Though you may build a single-page application in React, it is not a requirement. React can also be used for enhancing small parts of existing websites with additional interactivity. Code written in React can coexist peacefully with markup rendered on the server by something like PHP, or with other client-side libraries. In fact, this is exactly how React is being used at Facebook.
+यद्यपि आप React में सिंगल पेज एप्लिकेशन का निर्माण कर सकते हैं, यह कोई आवश्यकता नहीं है। React का उपयोग मौजूदा वेबसाइटों के छोटे हिस्सों को अतिरिक्त इंट्रक्टिविटी बढ़ाने के लिए भी किया जा सकता है। React में लिखा कोड सर्वर पर PHP, या अन्य क्लाइंट-साइड लाइब्रेरी जैसी किसी चीज़ के द्वारा रेंडर किए गए मार्कअप के साथ शांतिपूर्वक सहवास कर सकता है। वास्तव में, ठीक इसी तरह फेसबुक में React का उपयोग किया जा रहा है।
```

Comment:
पेज or prishth, use once of them throughout the document please?

Reply:
वेबसाइटों -> वेबसाइटs

Reply:
React का उपयोग मौजूदा वेबसाइटों के छोटे हिस्सों को अतिरिक्त इंट्रक्टिविटी बढ़ाने के लिए भी किया जा सकता है। does not seem right translation, please revisit

Reply:
next line is contextually incorrect as well

Reply:
> next line is contextually incorrect as well

यद्यपि आप React में सिंगल-पृष्ठ एप्लिकेशन का निर्माण कर सकते हैं,पर यह कोई आवश्यक नहीं है। React का उपयोग मौजूदा वेबसाइटस के छोटे हिस्सों में अतिरिक्त इंट्रक्टिविटी बढ़ाने के लिए भी किया जा सकता है। React में लिखा कोड PHP, या अन्य क्लाइंट-साइड लाइब्रेरीज के द्वारा सर्वर पर रेंडर किए गए मार्कअप के साथ शांतिपूर्वक सहवास कर सकता है। वास्तव में, फेसबुक में React का उपयोग ठीक इसी तरह किया जा रहा है।

Reply:
React में लिखा कोड PHP, या अन्य क्लाइंट-साइड लाइब्रेरीज के द्वारा सर्वर पर रेंडर किए गए मार्कअप के साथ शांतिपूर्वक सहवास कर सकता है। वास्तव में, फेसबुक में React का उपयोग ठीक इसी तरह किया जा रहा है।

 still contextually incorrect

Reply:
> React में लिखा कोड PHP, या अन्य क्लाइंट-साइड लाइब्रेरीज के द्वारा सर्वर पर रेंडर किए गए मार्कअप के साथ शांतिपूर्वक सहवास कर सकता है। वास्तव में, फेसबुक में React का उपयोग ठीक इसी तरह किया जा रहा है।
> 
> still contextually incorrect

React में लिखा गया कोड PHP, या अन्य क्लाइंट-साइड लाइब्रेरीज के द्वारा सर्वर पर रेंडर किए गए मार्कअप के साथ शांतिपूर्वक सहवास कर सकता है। फेसबुक में React का उपयोग ठीक इसी तरह किया जाता है।

Reply:
React में लिखा गया कोड PHP, या अन्य क्लाइंट-साइड लाइब्रेरीज के द्वारा सर्वर पर रेंडर किए गए मार्कअप के साथ शांतिपूर्वक सहवास कर सकता है। server rendering in client side libraries is still not a correct statement :(

Reply:
> React में लिखा गया कोड PHP, या अन्य क्लाइंट-साइड लाइब्रेरीज के द्वारा सर्वर पर रेंडर किए गए मार्कअप के साथ शांतिपूर्वक सहवास कर सकता है। server rendering in client side libraries is still not a correct statement :(

React में लिखा गया कोड  व PHP, या अन्य क्लाइंट-साइड लाइब्रेरीज के द्वारा रेंडर  किया गया  मार्कअप  शांतिपूर्वक  सर्वर पर सहवास कर सकते है

Reply:
React में लिखा गया कोड anya libraries ke saath render kiya ja sakta hai. React markup server rendered libraries jaise ki PHP ya anya client side libraries ke saath shaantipurwak render ho sakta hai

Reply:
> React में लिखा गया कोड anya libraries ke saath render kiya ja sakta hai. React markup server rendered libraries jaise ki PHP ya anya client side libraries ke saath shaantipurwak render ho sakta hai

 React में लिखा गया कोड अन्य लाइब्रेरीज के साथ रेंडर किया जा सकता है। React  मार्कअप सर्वर रेंडरड लाइब्रेरीज  जैसे की PHP या अन्य क्लाइंट-साइड लाइब्रेरीज के साथ शांतिपूर्वक रेंडर हो सकता है।